public/app/features/alerting/unified/api/alertRuleApi.ts

```diff
@@ -127,5 +128,9 @@ export const alertRuleApi = alertingApi.injectEndpoints({
         return groupRulesByFileName(response.data.groups, GRAFANA_RULES_SOURCE_NAME);
       },
     }),
+
+    exportRule: build.query<string, { uid: string; format: 'yaml' | 'json' }>({
+      query: ({ uid, format }) => ({ url: `${PROVISIONING_URL.replace('{UID}', uid)}?format=${format}` }),
```

Comment:
What do you think if instead of using the `PROVISIONING_URL` constant we added a new function like `getProvisioningUrl(ruleUid: string, format: 'yaml' | 'json')`? It could be safer than replacing string tokens